README.md

```diff
@@ -345,7 +345,7 @@ The exported CSV file columns contain the following user information and structu
 |jasonR@example.com|R|Jason| |
 
 The field **groups** contains a comma-separated list of all iAuditor groups the user is a member of. All fields are string values.
-Note: comma-separated group names are not supported. If a group name contains a comma, it will be listed as is.
+Note: comma-separated group names are not supported. If group names contains commas, it will be listed as is.
```

Comment:
Note: Group names that contain commas are not supported. If a group name contains a comma, it will be listed as is and no users will be assigned to it as a result.